bluebottle/funding/tests/test_admin.py

```diff
@@ -153,6 +154,18 @@ def test_donation_reward(self):
         self.assertTrue(second.title in response.content.decode('utf-8'))
         self.assertFalse(third.title in response.content.decode('utf-8'))
 
+    def test_contribution_date(self):
+        donation = DonorFactory.create(
+            activity=self.funding,
+            amount=Money(100, 'EUR')
+        )
+        donation.states.succeed(save=True)
+        self.client.force_login(self.superuser)
+        response = self.client.get(self.admin_url)
+        curr_time = now()
```

Comment:
You probably want to use this as `donation.created` too.

Reply:
Make sure to do the `self.client.get` after you set that value.
Also, you might want to use `self.app.get` from the webtest stack. You probably have to change the `TestCase` this extends or create a new `TestCase`. Webtest (the one you used in your other tests) has html searching features, which you can use to find the first column of the displayed table.